File: src/main/java/io/github/wntopia/gikipedia/server/GikipediaServerApplication.java
package io.github.wntopia.gikipedia.server;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;

@SpringBootApplication
@ConfigurationPropertiesScan
public class GikipediaServerApplication {

  static void main(String[] args) {
    SpringApplication.run(GikipediaServerApplication.class, args);
  }
}

File: src/main/kotlin/io/github/wntopia/gikipedia/server/GikipediaServerApplication.kt
package io.github.wntopia.gikipedia.server

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
class GikipediaServerApplication

fun main(args: Array<String>) {
    runApplication<GikipediaServerApplication>(*args)
}
